docs/getting_started/dashboard/work_with_dashboard.md

```diff
@@ -47,11 +47,11 @@ C. Buttons:
 |Redo|![Redo](dashboard_redo.png)|Redo latest change.|
 |Structure view|![Structure view](dashboard_structure_view.png)|Access Structure view toolbar.|
 |Dashboard blocks|![Dashboard blocks](dashboard_blocks.png)|Access Dashboard blocks toolbar.|
```

Comment:
```suggestion
|Structure view|![Structure view](dashboard_structure_view.png)|Access **Structure view** menu.|
|Dashboard blocks|![Dashboard blocks](dashboard_blocks.png)|Access **Dashboard blocks** menu.|
```

Also, in line 38 above, please change "toolbars" to "menus"

Reply:
Updated with 'toolbox'.